### PR TITLE
docs: add note

### PR DESCRIPTION
**Pull request by wizardofhogwarts**
#2872 and this pr could resolve the issue. The issue in #2857 will also get resolved.

### CODE REVIEW

File: src/content/concepts/loaders.md [hunk 1]
```diff
@@ -80,6 +81,7 @@ module.exports = {
 };
 ```
 
+> Note: For detailed information about [style-loader](/loaders/style-loader), [css-loader](/loaders/css-loader) and [sass-loader](/loaders/sass-loader), please visit the links.
```
Comment (EugeneHlushko) [inline src/content/concepts/loaders.md]: Move them to the further reading section

Comment (wizardofhogwarts) [reply src/content/concepts/loaders.md]: @EugeneHlushko where is that section?

Comment (EugeneHlushko) [reply src/content/concepts/loaders.md]: Its here: https://webpack.js.org/contribute/writers-guide#yaml-frontmatter

**Comment by montogeek** [comment]
This is not necessary, real solution is in https://github.com/webpack/webpack.js.org/pull/2857

**Comment by wizardofhogwarts** [comment]
> This is not necessary, real solution is in #2857, we need to rename the file and update the Codeblock language.

https://github.com/webpack/webpack.js.org/pull/2857#issuecomment-471999756 suggested removing the links in the context of `loaders.md`. 
https://github.com/webpack/webpack.js.org/pull/2872#pullrequestreview-216517445 shows my comment on #2872 
@montogeek As #2857 and #2872 appears to be same. So, we should merge #2857 with #2889 (this pr).

File: src/content/concepts/loaders.md (hunk 1 above)
Comment (wizardofhogwarts) [reply src/content/concepts/loaders.md]: @EugeneHlushko thanks

Comment (wizardofhogwarts) [reply src/content/concepts/loaders.md]: @EugeneHlushko done with the further reading section

**Comment by EugeneHlushko** [comment]
I must admit, now i fully understand the issue, lets rename the file as suggested in #2857, this will be redundant :/

**Comment by wizardofhogwarts** [comment]
> I must admit, now i fully understand the issue, lets rename the file as suggested in #2857, this will be redundant :/

should I change it to .mdx file?

**Comment by montogeek** [comment]
Yes, rename it to `.mdx` and update Code block to ````js-with-links-details`

**Comment by EugeneHlushko** [comment]
@montogeek want to test it locally or should i?

**Comment by montogeek** [comment]
I will later today

**Comment by montogeek** [comment]
This is how it looks:
image

**Comment by wizardofhogwarts** [comment]
> This is how it looks:
> image

Output looks fine. I think i should remove square brackets from each loader's name.

File: src/content/concepts/loaders.mdx [hunk 2]
```diff
@@ -12,6 +12,14 @@ contributors:
   - byzyk
   - debs-obrien
   - EugeneHlushko
+  - wizardofhogwarts
+related:
```
Comment (EugeneHlushko) [inline src/content/concepts/loaders.mdx]: Since you've fixed the links, this section is not needed anymore :(

Comment (wizardofhogwarts) [reply src/content/concepts/loaders.mdx]: @EugeneHlushko can we just keep references at one place for further reading? What do you think?

Comment (EugeneHlushko) [reply src/content/concepts/loaders.mdx]: IMO, links had good placement in their context, as a fix to non working block it looked like a good idea to me, right now, it looks wrong as from a guide about loaders you link to three specific ones as a read more. Despite us having a link to loaders in the header 
Screen Shot 2019-03-25 at 11 09 37 AM

Comment (wizardofhogwarts) [reply src/content/concepts/loaders.mdx]: @EugeneHlushko related section removed

**Comment by montogeek** [comment]
Thanks!

**Pull request merged**